.gitignore

```diff
@@ -0,0 +1,6 @@
+.idea/
+asset/node_modules/
+theme/css/
+theme/img/
+theme/js/
```

Comment:
yes, all repos build assets as part of deployment - as it was suggested in https://github.com/zendframework/zf-mkdoc-theme/issues/10#issuecomment-227377075

BUT ! we don't need any changes in travis configuration for each repo! Please have a look on changes in `build.sh` :)

I'm happy to change it, to build these asset once here if you'd prefer it.